src/server/node_manager.c

```diff
@@ -529,13 +529,20 @@ set_all_state(mominfo_t *pmom, int do_set, unsigned long bits, char *txt,
 	attribute	*pat;
 	int		nchild;
 	unsigned long	inuse_flag = 0;
+	char local_log_buffer[LOG_BUF_SIZE];
 
 	if (do_set) { /* STALE is not meaning in the state of the Mom, don't set it */
 		psvrmom->msr_state |= (bits & ~INUSE_STALE);
 	} else {
 		psvrmom->msr_state &= ~bits;
 	}
 
+	local_log_buffer[LOG_BUF_SIZE-1] = '\0';
+	snprintf(local_log_buffer, LOG_BUF_SIZE-1, "set_all_state;"
+		"txt=%s mi_modtime=%ld", txt, pmom->mi_modtime);
+	log_event(PBSEVENT_DEBUG2, PBS_EVENTCLASS_NODE, LOG_INFO,
```

Comment:
Please use log_eventf() so the need for local_log_buffer can be taken out.

Reply:
Done